packages/module/patternfly-docs/content/examples/propertiesSidePanel.css

```diff
@@ -1,21 +1,18 @@
 .properties-side-panel-pf {
```

Comment:
mentioned this above but all of the styles in this stylesheet come with the `<DescriptionList>` component